core/assets/bundles/bundle.properties

```diff
@@ -375,9 +375,9 @@ zone.overgrowth.description = This area is overgrown, closer to the source of th
 zone.tarFields.description = The outskirts of an oil production zone, between the mountains and desert. One of the few areas with usable tar reserves.\nAlthough abandoned, this area has some dangerous enemy forces nearby. Do not underestimate them.\n\n[lightgray]Research oil processing technology if possible.
 zone.desolateRift.description = An extremely dangerous zone. Plentiful resources, but little space. High risk of destruction. Leave as soon as possible. Do not be fooled by the long spacing between enemy attacks.
 zone.nuclearComplex.description = A former facility for the production and processing of thorium, reduced to ruins.\n[lightgray]Research the thorium and its many uses.\n\nThe enemy is present here in great numbers, constantly scouting for attackers.
-zone.fungalPass.description = A transition area between high mountains and lower, spore-ridden lands. A small enemy reconnaissance base is located here.\nDestroy it.\nUse Dagger and Crawler units. Take out the two cores.
+zone.fungalPass.description = A transition area between high mountains and lower, spore-ridden lands. A small enemy reconnaissance base is located here.\nDestroy it.\nUse Dagger and Crawler units. Take out the two cores. 
 zone.impact0078.description = <insert description here>
-zone.crags.description = <insert description here>
+zone.crags.description = A wasteland ridden with cracks from old-war impact weapons.
```

Comment:
"A wasteland ridden with cracks from old impact weapons." "old-war" seems a little off.

Reply:
okie
ill try to find something else
do you have any other ideas?